.travis.yml

```
@@ -33,20 +41,39 @@ script:
 jobs:
   fast_finish: true
   allow_failures:
+    - php: 7.3
     - php: nightly
   include:
+    - php: 7
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7
+    - php: 7.1
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.1
+    - php: 7.2
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.2
+    - php: 7.3
+      env: COMPOSER_ARGS="--prefer-lowest"
+    - php: 7.3
+    - php: nightly
+      env: COMPOSER_ARGS="--prefer-lowest"
```

Will want to ignore platform requirements for 7.3 and nightly, otherwise composer won't install dependencies.

```suggestion
      env: COMPOSER_ARGS="--prefer-lowest --ignore-platform-reqs"
```